src/app-gocardless/banks/ssk-dusseldorf-dussdeddxxx.js

```diff
@@ -0,0 +1,53 @@
+import Fallback from './integration-bank.js';
+
+import { formatPayeeName } from '../../util/payee-name.js';
+
+/** @type {import('./bank.interface.js').IBank} */
+export default {
+  ...Fallback,
+
+  institutionIds: ['SSK_DUSSELDORF_DUSSDEDDXXX'],
+
+  accessValidForDays: 90,
```

Comment:
This can be removed as it's the default, it'll make it easier to support longer validity lengths in the future

```suggestion
```